cluster/src/table_manager.rs

```diff
@@ -74,26 +87,27 @@ impl TableManager {
     }
 }
 
-struct TableManagerInner {
-    shards_info: Vec<ShardInfo>,
-    schemas_info: HashMap<String, SchemaInfo>,
-    // schema_name -> table_name -> (shard_info, table_info)
-    tables: BTreeMap<String, BTreeMap<String, (ShardInfo, TableInfo)>>,
+#[derive(Debug, Default)]
+struct Inner {
+    shard_infos: HashMap<ShardId, ShardInfo>,
+    schema_infos: HashMap<String, SchemaInfo>,
+    tables: BTreeMap<SchemaName, BTreeMap<TableName, ShardTableInfo>>,
```

Comment:
This looks a bit unreasonable to me. We will need to query table infos by either schema name or shard id, but not one on top of another.

Reply:
In the future, let's decide how to re-design the mapping when this module is integrated.

Reply:
👌